python/ray/data/_internal/execution/streaming_executor_state.py

```diff
@@ -138,7 +138,7 @@ def initialize_progress_bars(self, index: int, verbose_progress: bool) -> int:
         enabled = verbose_progress or is_all_to_all
         self.progress_bar = ProgressBar(
             "- " + self.op.name,
-            self.op.num_outputs_total() or 1,
+            self.op.num_outputs_total(),
```

Comment:
nice! can we verify (unit test is best, if not, we can run code manually and examine logs) that we can do a similar change for the global progress bar, now that we are properly getting the total num outputs? thanks!
https://github.com/ray-project/ray/blob/master/python/ray/data/_internal/execution/streaming_executor.py#L107

Reply:
Whoops I missed that part. I don't see why we can't do the same for that as well. It's pretty much just taking the last operator and repeating the logic we have for the opstate progress bars. Should the existing tests already cover this?